### ISSUE TITLE
Ctrl+Space isn't working in multiselect with autocomplete

### ISSUE THREAD

**Issue by noxify** [TopCli/prompts#91]
Hi,

first of all - Thanks for this library. 

I'm currently testing some stuff and while testing it with the following snippet:

```ts
import { multiselect } from "@topcli/prompts"

const remoteSchemas = Array.from(Array(50).keys()).map((val) => {
  return { value: `tablename-${val}`, label: `Table ${val}` }
})

const selectedSchema = await multiselect("Choose a schema", {
  choices: remoteSchemas,
  autocomplete: true,
})
```

it seems that `Ctrl+Space` isn't working.

All other commands seems to work as expected.

--- 

Tested with an Intel MBP PRO 13" - Lastest OSX - Terminal via zsh

--- 

I assume that based on the changes in https://github.com/TopCli/prompts/pull/62 it is the expected behavior.

Question:
* Is my assumptions correct?
* If yes:
  * Should we remove the `<Ctrl+Space>` from the hint?
  * Or should we (re-)add the toggle via **`Ctrl`**+`Space`  ( like it's already implemented for the toggle all ( via `Ctrl+A` )

If I can help, please let me know and I will try make the required changes.
( haven't looked in detail into the code, but based on what I have seen, I think this is should be to hard to implement )

**Comment by PierreDemailly** [comment]
Hi @noxify, thanks for reporting.
Indeed, the hint is wrong. This issue has been fixed in a previous PR https://github.com/TopCli/prompts/pull/86/files
But it's not released yet, I'll release a minor this evening.

FYI Ctrl+Space is not supported -> https://github.com/nodejs/node/issues/2890
Same for Ctrl + Backspace, this is why I closed #68

**Comment by noxify** [comment]
Perfect - thanks for the clarification and the quick response.
Will close this issue since it's already fixed via #86
